### ISSUE TITLE
error gym

### ISSUE THREAD

**Issue by ZenGrasper** [PKU-EPIC/UniDexGrasp2#4]
Hi, when run bash script/run_train_ppo_state.sh
Importing module 'gym_38' (/home/XXX/Downloads/Dexgrasp/IsaacGym_Preview_4_Package/isaacgym/python/isaacgym/_bindings/linux-x86_64/gym_38.so)
Setting GYM_USD_PLUG_INFO_PATH to /home/XXX/Downloads/Dexgrasp/IsaacGym_Preview_4_Package/isaacgym/python/isaacgym/_bindings/linux-x86_64/usd/plugInfo.json
PyTorch version 1.13.0+cu117
Device count 1
/home/XXX/Downloads/Dexgrasp/IsaacGym_Preview_4_Package/isaacgym/python/isaacgym/_bindings/src/gymtorch
Using /home/XXX/.cache/torch_extensions/py38_cu117 as PyTorch extensions root...
Emitting ninja build file /home/XXX/.cache/torch_extensions/py38_cu117/gymtorch/build.ninja...
Building extension module gymtorch...
Allowing ninja to set a default number of workers... (overridable by setting the environment variable MAX_JOBS=N)
ninja: no work to do.
Loading extension module gymtorch...
Setting seed: 0
Algorithm:  ppo
Python
Averaging factor:  0.01
Obs type: full_state
self.device_id: 0
self.graphics_device_id 0
Not connected to PVD
+++ Using GPU PhysX
Physics Engine: PhysX
Physics Device: cuda:0
GPU Pipeline: enabled
**[Error] [carb.gym.plugin] Graphics is nullptr in GymCreateTextureFromFile**
/home/XXX/UniDexGrasp2/dexgrasp/tasks/shadow_hand_grasp.py:243: DeprecationWarning: an integer is required (got type isaacgym._bindings.linux-x86_64.gym_38.DofDriveMode).  Implicit conversion to integers using __int__ is deprecated, and may be removed in a future version of Python.
  asset_options.default_dof_drive_mode = gymapi.DOF_MODE_NONE
JointSpec type free not yet supported!
self.num_shadow_hand_bodies:  24
self.num_shadow_hand_shapes:  20

...

The code can be trained, but there is no GYM interface. Will the Gym interface appear after the code runs?
My conda environment : CUDA11.7，Python3.8，isacc gym preview 4 ...

**Comment by wkwan7** [comment]
Hi, after the installation of IsaacGym, you need to ensure that IsaacGym works on your system by running one of the examples from the `python/examples directory` in IsaacGym, like` joint_monkey.py`. Please follow troubleshooting steps described in the Isaac Gym Preview Release 3/4 install instructions if you have any trouble running the samples. The code should work if the example in IsaacGym works :)

**Comment by EchoSilverLining** [comment]
Hello, thank you for your  work, and thank you for your reply. I reinstalled IsaacGym again. When running the IsaacGym example, it runs smoothly and the visualization interface appears as expected. But, when run bash script/run_train_ppo_state.sh, the **"[Error] [carb.gym.plugin] Graphics is nullptr in GymCreateTextureFromFile"** error still persists.

**Comment by wkwan7** [comment]
Hi, can you try without the first line or without `--headless` in the script? 
Not sure if this is the issue. The script should work if the IsaacGym example works.

**Comment by EchoSilverLining** [comment]
Thanks again for your help. After removing '--headless', Gym can display properly. Thank you so much!
